### ISSUE TITLE
Tracking: track whether or not it is the new flow

### ISSUE THREAD

**Issue by dmallory42** [Automattic/woocommerce-payments#6710]
### Description

We found when trying to create Tracking funnels that it is hard to narrow down the cohort to only those who went through the new UX flow (vs the old one). We can mitigate this by adding a parameter to the `wcadmin_wcpay_connect_account_clicked`, something like `is_new_flow` in order to segment appropriately from the beginning of the flow.

### Acceptance criteria

* We have a way to identify which flow was being used in the `wcadmin_wcpay_connect_account_clicked` event in order to better segment our results.

**Comment by brandonwanA8C** [comment]
@dmallory42 
is the use case for this issue to track data for 1) users who started with old/existing (pre-PO) UX vs. 2) users who started with new-TOFU (as a result of the PO project)?

**Comment by dmallory42** [comment]
Hi @brandonwanA8C! 

We found that even though we have added events for all stages of the new flow, we were struggling to identify which users from the first step (the `wcpay_connect_account_clicked` event when they start the onboarding) were sent to the new TOFU-UX vs the old experience. 

This change will allow us to segment those users in Tracks, and as you said allows us to track and compare the two groups more easily.
